src/v2/Apps/Artist/artistRoutes.tsx

```diff
@@ -120,84 +120,42 @@ export const artistRoutes: RouteConfig[] = [
         prepare: () => {
           WorksForSaleRoute.preload()
         },
-        prepareVariables: (params, props) => {
+        prepareVariables: ({ artistID }, props) => {
           // FIXME: The initial render includes `location` in props, but subsequent
           // renders (such as tabbing back to this route in your browser) will not.
           const filterStateFromUrl = props.location ? props.location.query : {}
 
           const filterParams = {
             ...initialArtworkFilterState,
             ...paramsToCamelCase(filterStateFromUrl),
-            ...params,
           }
 
-          filterParams.hasFilter = Object.entries(filterParams).some(
-            ([k, v]: [keyof ArtworkFilters, any]) => {
-              return !isDefaultFilter(k, v)
-            }
-          )
+          // filterParams.hasFilter = Object.entries(filterParams).some(
+          //   ([k, v]: [keyof ArtworkFilters, any]) => {
+          //     return !isDefaultFilter(k, v)
+          //   }
+          // )
```

Comment:
What's this?

Reply:
https://github.com/artsy/force/pull/7151/files?file-filters%5B%5D=.ts&file-filters%5B%5D=.tsx#r594606837

Nothing/unclear! Commenting it out for now.